openmetadata-ui/src/main/resources/ui/src/pages/CustomPropertiesPageV1/CustomPropertiesPageV1.tsx

```diff
@@ -126,7 +126,7 @@ const CustomEntityDetailV1 = () => {
         setIsButtonLoading(false);
       }
     },
-    [selectedEntityTypeDetail.id]
+    [selectedEntityTypeDetail]
```

Comment:
Why we change this? Can you add details of it?

Reply:
So we are using selectedEntityTypeDetail, as well inside the function, and the other data was not updating due to the specified dependancy